base/applications/osk/main.c

```diff
@@ -567,6 +625,17 @@ INT_PTR APIENTRY OSK_DlgProc(HWND hDlg, UINT msg, WPARAM wParam, LPARAM lParam)
             }
             break;
 
+        case WM_THEMECHANGED:
+            /*
+                A theme change has occurred. The hTheme member holds the previous
+                theme style and preserving it would cause a lot of issues. Therefore,
+                close the handle with the previous theme and initiate a new one for the
+                changed theme.
+            */
+            CloseThemeData(Globals.hTheme);
+            Globals.hTheme = OpenThemeData(hDlg, L"Button");
```

Comment:
This is incorrect, you should pass a handle to a Button window here or you could end up with unpredictable behaviour.